client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.controller.js

```diff
@@ -8,7 +8,7 @@ angular.module("managerApp").controller("TelecomTelephonyLinePhoneOrderCtrl", fu
 
     function fetchOfferPhones (offer) {
         return OvhApiTelephony.Lexi().getLineOfferPhones({
-            country: "fr",
+            country: _.get(self.user, "country", "fr").toLowerCase(),
```

Comment:
May be dangerous if the user specify an address with a country that is not in `telephony.NumberCountryEnum`. e.g. an enterprise coming from Poland and have a branch in France that needs telephony services. 
I think you should detect if country is in the `telephony.NumberCountryEnum` and if not you should select "FR".